root/static/scripts/release-editor/dialogs.js

```diff
@@ -124,6 +128,29 @@ class SearchResult {
             track.artistCredit = [{ name: track.artist }];
         }
     }
+
+    link() {
```

Comment:
Here's one of the strings that changed. See https://github.com/metabrainz/musicbrainz-server/pull/704/files#diff-6edb40ffe5e08874aaa80abd6c4d1045L123

I think it's an improvement since I removed the `by {artist}` string (concatenated to the release name) and reused an existing `{release} by {artist}` string. Not sure if the way I handle the format after the release name is the best.